File: src/data/datatypes/phone.ts
import {Entity, PrimaryGeneratedColumn, Column, ManyToOne} from "typeorm";
import Costumer from "./costumer";

@Entity()
export class Phone {

    @PrimaryGeneratedColumn()
    id: number;

    @Column({nullable:false})
    ddd: string;

    @Column({nullable:false})
    number: string;

    @ManyToOne(()=> Costumer, costumer => costumer.phones)
    costumer:Costumer;

}

File: Structures/Models/Phone.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
namespace Comies.Structures.Models
{
    public class Phone : Entity
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }

        [MaxLength(2)]
        public string DDD { get; set; }

        [MaxLength(9)]
        public string Number { get; set; }
        public Guid? CostumerId { get; set; }
        public virtual Costumer Costumer { get; set; }
    }
}